lib/azure.js

```diff
@@ -1327,6 +1327,73 @@ exports.createInsightsManagementClient = function (credentials, subscriptionId,
+ *
+ * @param {Array} [options.filters] - Filters to be added to the request pipeline
+ *
+ * @param {object} [options.requestOptions] - Options for the underlying request object
+ * {@link https://github.com/request/request#requestoptions-callback Options doc}
+ *
+ * @param {boolean} [options.noRetryPolicy] - If set to true, turn off default retry policy
+ *
+ * @param {string} [options.acceptLanguage] - Gets or sets the preferred language for the response.
+ *
+ * @param {number} [options.longRunningOperationRetryTimeout] - Gets or sets the retry timeout in seconds for Long Running Operations. Default value is 30.
+ *
+ * @param {boolean} [options.generateClientRequestId] - When set to true a unique x-ms-client-request-id value is generated and included in each request. Default is true.
+ *
+ * @return {MonitorManagementClient}                A new MonitorManagementClient object.
+*/
+exports.createMonitorManagementClient = function (credentials, subscriptionId, options) {
+  const azureARMMonitor = require('azure-arm-monitor');
+  return new azureARMMonitor.MonitorManagementClient(credentials, subscriptionId, null, options);
+};
+
+/**
+ * @class
+ * Initializes a new instance of the MonitorClient class.
+ * @constructor
+ *
+ * @param {credentials} credentials - Credentials needed for the client to connect to Azure.
+ *
+ * @param {string} subscriptionId - The Azure subscription Id.
+ *
+ * @param {object} [options] - The parameter options
+ *
+ * @param {Array} [options.filters] - Filters to be added to the request pipeline
+ *
+ * @param {object} [options.requestOptions] - Options for the underlying request object
+ * {@link https://github.com/request/request#requestoptions-callback Options doc}
+ *
+ * @param {boolean} [options.noRetryPolicy] - If set to true, turn off default retry policy
+ *
+ * @param {string} [options.acceptLanguage] - Gets or sets the preferred language for the response.
+ *
+ * @param {number} [options.longRunningOperationRetryTimeout] - Gets or sets the retry timeout in seconds for Long Running Operations. Default value is 30.
+ *
+ * @param {boolean} [options.generateClientRequestId] - When set to true a unique x-ms-client-request-id value is generated and included in each request. Default is true.
+ *
+ * @return {InsightsClient}                          A new InsightsClient object.
+*/
+exports.createMonitorClient = function (credentials, subscriptionId, options) {
+  const azureMonitor = require('azure-monitor');
+  return new azureMonitor.MonitorClient(credentials, subscriptionId, null, options);
+};
+
+var azureARMDatalakeAnalytics = require('azure-arm-datalake-analytics');
+exports.DataLakeAnalyticsAccountClient = azureARMDatalakeAnalytics.DataLakeAnalyticsAccountClient;
+exports.DataLakeAnalyticsJobClient = azureARMDatalakeAnalytics.DataLakeAnalyticsJobClient;
+exports.DataLakeAnalyticsCatalogClient = azureARMDatalakeAnalytics.DataLakeAnalyticsCatalogClient;
```

Comment:
1. why changes to datalake? This PR is about monitor isn't it?
2. follow the pattern of other factory methods createXXXClient where it lazy loads the dependent package